Docs: New Guide for adding art/sprites to Stonesense (#5207)
DFHack/dfhack: docs/plugins/stonesense.rst

Stonesense requires working graphics acceleration, and we recommend
at least a dual core CPU to avoid slowing down your game of DF.

Yellow Cubes and Missing Sprites
--------------------------------

> If you are seeing yellow cubes in Stonesense, that is simply things we don't have sprites for.

Reviewer: ```suggestion
If you are seeing yellow cubes in Stonesense, then there is something on the map that
Stonesense does not have a sprite for.
```

.. figure:: ../images/stonesense-yellowcubes.png
   :align: center

   An example of the yellow cubes.

If you would like to help us in fixing this, there are 2 things you can do:

> * Make an issue on Github with what item is missing and pictures of what it looks like in DF

Reviewer: ```suggestion
* Make an issue on GitHub with what item is missing and pictures of what it looks like in DF.
```

Reviewer: should include a hyperlink to where they should file the issue

> * Create the art yourself. For help with this, please see the `stonesense-art-guide`

Reviewer: ```suggestion
* Create the art yourself. For help with this, please see the `stonesense-art-guide`.
```

Stonesense requires working graphics acceleration, and we recommend
at least a dual core CPU to avoid slowing down your game of DF.

Yellow Cubes and Missing Sprites
--------------------------------
If you are seeing yellow cubes in Stonesense, then there is something on the map that
Stonesense does not have a sprite for.

.. figure:: ../images/stonesense-yellowcubes.png
   :align: center

   An example of the yellow cubes.

If you would like to help us in fixing this, there are 2 things you can do:

> * Make an issue on `Github <https://github.com/DFHack/stonesense/issues>`_ with what item is missing and pictures of what it looks like in DF

Reviewer: I think the continuation line is two spaces. Might be four -- please double check that this renders properly
